radosgw-agent-pull-requests/build/build

```diff
@@ -1,22 +1,17 @@
 #!/bin/bash
 
-set -ex
+# the following two methods exist in scripts/build_utils.sh
+pkgs=( "ansible" "tox" )
+install_python_packages "pkgs[@]"
 
-# Create the virtualenv
-virtualenv venv
-. venv/bin/activate
 
-# Define and ensure the PIP cache
-PIP_SDIST_INDEX="$HOME/.cache/pip"
-mkdir -p $PIP_SDIST_INDEX
-
-# Install the package by trying with the cache first, otherwise doing a download only, and then
-# trying to install from the cache again.
-if ! venv/bin/pip install --find-links="file://$PIP_SDIST_INDEX" --no-index tox; then
-    venv/bin/pip install --download-directory="$PIP_SDIST_INDEX" tox
-    venv/bin/pip install --find-links="file://$PIP_SDIST_INDEX" --no-index tox
-fi
+# run ansible to get this current host to meet our requirements, specifying
+# a local connection and 'localhost' as the host where to execute. This might look
+# odd because we are using ceph-deploy playbooks. But the requirements are the same
+cd "$WORKSPACE/ceph-build/ceph-deploy-pull-requests/setup/playbooks"
+$VENV/ansible-playbook -i "localhost," -c local setup.yml
```

Comment:
So you're doing this so that python 2.6 will be installed? If so, a comment might help clarify this for future readers.